utils/githooks/find_base.sh

```diff
@@ -23,9 +30,27 @@ if command -v gh > /dev/null 2>&1; then
         fi
     fi
 else
-    # With no 'gh' command installed then check against origin/master.
+    # With no 'gh' command installed, use the "closest" branch as the target,
+    # calculated as the sum of the commits this branch is ahead and behind.
+    # check master, then current release branches, then current feature branches.
     # shellcheck disable=SC2034
-    TARGET="$ORIGIN"/master
+    all_bases=("master")
+    read -a feature_branches <<< $(git branch --list -r "$ORIGIN/release/*" \
+                                   | grep -oE "release/2\.[4-9]+.*|release/[3-9]+.*")
+    all_bases+=(${feature_branches[@]})
```

Comment:
(lint) Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a. [SC2206]